plug-ins/dns_spoof/dns_spoof.c

```diff
@@ -545,12 +567,15 @@ static int prepare_dns_reply(u_char *data, const char *name, int type, int *dns_
          len = 12 + IP_ADDR_LEN;
          SAFE_CALLOC(answer, len, sizeof(u_char));
          p = answer;
-      
+
+         /* make TTL big-endian to work with DNS */
+         ttl_bige = htonl(ttl);
```

Comment:
TTL is used later to log when DNS replies are sppofed and print out TTL, it may be simpler to use a separate variable rather than a call to `ntohl()` (in case system is little-endian). Otherwise it may be possible to simply use `htonl(ttl)`in the memcpy call rather than `ttl_bige`